easy-spinnaker/config_hal.sh

```diff
@@ -0,0 +1,65 @@
+# OpsMx 
+# Hal Configuration Script
+# Please note that this script should be run ONLY ONCE
+#######################################################################################################
+# Update this information based on your environment and uncomment the appropriate lines
+# Instructions for creating guthub-token can be found at https://github.com/settings/tokens
+# and https://www.spinnaker.io/setup/artifacts/github
+#######################################################################################################
```

Comment:
May be we can use conditional checking to see if these variables are set and then take action or not rather than having the user to comment and uncomment. Also, use of environment variables to drive these would be good. 

Not necessary to incorporate this but may be something for future.